benchmarks/run-benchmark.sh

```diff
@@ -6,11 +6,52 @@ set -euo pipefail
 BENCH_NAME=$1
 FILE=$(mktemp)
 
-if ! type "drun" > /dev/null; then
-  echo "drun is not installed. Please add drun to your path from commit d35535c96184be039aaa31f68b48bbe45909494e."
-  exit 1
+get_didc_release(){
+  OS=$(uname)
+  ARCH=$(uname -m)
+  if [ "$OS" == "Darwin" ] && [ "$ARCH" == "arm64" ]; then
+    # Apple sillicon
+    wget -O didc https://github.com/dfinity/candid/releases/download/2023-07-25/didc-macos
+  elif [ "$OS" == "Linux" ] && [ "$ARCH" == "x86_64" ]; then
+      # Linux x86 64bit
+      wget -O didc https://github.com/dfinity/candid/releases/download/2023-07-25/didc-linux64
+    else
+      echo "Unsupported machine"
+      EXIT SIGINT
+  fi
+}
+
+install_drun(){
+  URL="https://github.com/dfinity/ic/releases/download/release-2023-09-27_23-01%2Bquic/drun-x86_64-${OS}.gz"
+  wget -O "drun.gz" "${URL}"
+  gzip -fd drun.gz
+  chmod +x drun
+}
+
+get_correct_drun_release() {
+  OS=$(uname | tr '[:upper:]' '[:lower:]')
+  
+  if ! type "drun" > /dev/null; then
+    install_drun
+  else 
+    DRUN_LOCATION=$(type "drun" | awk '{print $3}')
+    DRUN_SHA=$(shasum -a 256 "$DRUN_LOCATION" | awk '{ print $1 }')
+    # Check if drun exists and if the correct version is used.
+    if ! [[ "$OS" == "linux" && "$DRUN_SHA" == "7bf08d5f1c1a7cd44f62c03f8554f07aa2430eb3ae81c7c0a143a68ff52dc7f7" ]]; then
```

Comment:
Let's move these constants to the top of the file please so it's clearer whenever we want to update these versions